src/styles/blocks/button-buy.scss

```diff
@@ -14,6 +14,7 @@
   background-color: $main-color;
   border: 1px solid $main-color;
   border-radius: 5px;
+  @include transition400ms(all);
```

Comment:
indicate which properties you want to use transition for. Don't use transition all

Reply:
Ok, thank you